cloud/deployment/deployment.go

```diff
@@ -335,6 +341,7 @@ func Create(name, workspaceID, description, clusterID, runtimeVersion, dagDeploy
 				Region:               &region,
 				IsDagDeployEnabled:   dagDeployEnabled,
 				IsHighAvailability:   highAvailabilityValue,
+				IsDevelopmentMode:    &developmentModeValue,
```

Comment:
Should this be?

```suggestion
				IsDevelopmentMode:    developmentModeValue,
```

Reply:
No, `IsDevelopmentMode` is a boolean pointer, because we wanted to have this field as optional. We don't want to introduce a breaking change to the platform api.

Reply:
is IsDevelopmentMode always returned by the API in this situation. If not panic errors could arise